src/components/atomic/molecules/Navigation.tsx

```diff
@@ -7,17 +7,27 @@ import { Link, useLocation } from "react-router-dom";
 import { Menu, Space } from "antd";
 import { NETWORKS, SUPPORTED_NETWORKS } from "../../../config/network";
 import { RiSafe2Line } from "react-icons/ri";
-import { useBreakpoints, useChainId, useRequestChangeNetworkCallback, useTranslator } from "hooks";
+import {
+  useBreakpoints,
+  useChainId,
+  useRequestChangeNetworkCallback,
+  useTranslator,
+} from "hooks";
 import { useMemo } from "react";
 import Icon from "@ant-design/icons";
 
 const NETWORKS_BY_ID: Record<number, any> = {
-  '1': NETWORKS.mainnet,
-  '137': NETWORKS.polygon
-}
+  "1": NETWORKS.mainnet,
+  "137": NETWORKS.polygon,
+};
 
-function NetworkIcon({ chainId }: {chainId: number}) {
-  return <img style={{ height: '2em'}} src={require(`images/${NETWORKS_BY_ID[chainId].icon}`).default} />
+function NetworkIcon({ chainId }: { chainId: number }) {
+  return (
+    <img
+      style={{ height: "2em" }}
```

Comment:
Storing component `style` values in an object-type variable, defined outside of the component, may make managing styles and readability easier.

```
const styles = {
  img: {
    height: '2em',
  },
};

function NetworkIcon({ chainId }: { chainId: number }) {
  return (
    <img
      style={styles.img}
...
```

Reply:
We'll be switching to styled-components and MUI, so we'll have this level of management upon integration - although there will be specific instances where we will need to directly inject params into the styles prop. 

Lots of pruning and refactoring to be before then though!